.github/workflows/ci.yml

```diff
@@ -215,7 +215,7 @@ jobs:
         mkdir ./build
         mkdir ./install
     - name: config
-      run: ../config --banner=Configured enable-fips --strict-warnings --prefix=$(cd ../install; pwd) && perl configdata.pm --dump
```

Comment:
I've done #16095 which enables the ACVP tests when FIPS is enabled which removes the need for this change.  Stuff like this should be automatic not manually enabled IMO.

Reply:
I'll merge this anyway. You can revert this hunk in #16095 if you want.